README.md

```diff
@@ -89,6 +89,20 @@ npm start
 ]
 ```
 
+* `CONNECTOR_ROUTES` (default: `[]`) Explicitly add routes to the connector. If `prefix` is the most specific
+  route name that matches a destination, then the payment will be forwarded to `connector`. ex:
+```js
+[
+  {
+    "prefix": "", // matches any destination
+    "connector": "ilpdemo.red.connie"
```

Comment:
I don't think the connector should assume that the ledger prefix of `"ilpdemo.red.connie"` is `"ilpdemo.red."`. I'd suggest renaming the `connector` field to `connectorAccount`, and adding a `connectorLedger` field as well.